openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -116,11 +119,45 @@ def _create_jwt(
         'filters': filters or [],
         'is_restricted': is_restricted,
         'email_verified': user.is_active,
+        'roles': _get_role_auth_claim_for_user(user),
     }
     payload.update(additional_claims or {})
     _update_from_additional_handlers(payload, user, scopes)
     return _encode_and_sign(payload, use_asymmetric_key, secret)
 
+def _get_role_auth_claim_for_user(user):
```

Comment:
i think when we were talking with Nimisha about the edx-rbac package, we talked bout having this be a util function in edx-rbac. i imagined we would use settings to indicate what the system wide user role classes were and pull from those. this means we'll have to add a get_context method to the top level class, so the same api for getting the context can be used across all models.